transformations/swap_synonyms/README.md

```diff
@@ -0,0 +1,68 @@
+# Butter Fingers Perturbation 🦎  + ⌨️ → 🐍
```

Comment:
I guess you planned to add swap_synonyms here?

Reply:
Yes. It has since been deleted.